src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/CatalogueSolutions.feature

```diff
@@ -446,10 +432,10 @@ Scenario Outline: Catalogue Solutions - edit price screen - Flat price Go back a
     And the Catalogue Solution is saved in the DB
 
     Examples:
-        | ProvisioningType |
-        #| declarative      |
-        #| variable         |
-        | per patient      |
+        | ProvisioningType | quantityLabel |
+        # | declarative   |  |
+        # | variable      |  |
```

Comment:
I guess you're waiting for the relevant FE work to be completed and will gradually reinstate the commented out provisioning types?

Reply:
Indeed that is the plan